docs/quick-start/project-structure.md

```diff
@@ -0,0 +1,197 @@
+
+```
+cd projects
+react-native init VanillaProject
+cd VanillaProject
+```
+
+Your new React Native project should have a file structure similar to the following:
+
+```
+VanillaProject
+├── __tests__
+│   ├── index.android.js
+│   └── index.ios.js
+├── android
+│   ├── app
+│   ├── build.gradle
+│   ├── gradle
+│   ├── gradle.properties
+│   ├── gradlew
+│   ├── gradlew.bat
+│   ├── keystores
+│   └── settings.gradle
+├── app.json
+├── index.android.js
+├── index.ios.js
+├── ios
+│   ├── VanillaProject
+│   ├── VanillaProject-tvOS
+│   ├── VanillaProject-tvOSTests
+│   ├── VanillaProject.xcodeproj
+│   ├── VanillaProjectTests
+│   └── build
+└── package.json
+```
+
+Now create a new Ignite project and lets compare the folder structure:
+
+Note: TODO: Maybe like what options we select when creating a new Ignite project?
+
+```
+cd ..
+ignite new IgniteProject
+cd IgniteProject
+```
+
+The Ignite project's structure will look similar to this:
+
+```
+IgniteProject
+├── App
+│   ├── Components
+│   ├── Config
+│   ├── Containers
+│   ├── Fixtures
+│   ├── Images
+│   ├── Lib
+│   ├── Navigation
+│   ├── Redux
+│   ├── Sagas
+│   ├── Services
+│   ├── Themes
+│   └── Transforms
+├── README.md
+├── __tests__
+│   ├── index.android.js
+│   └── index.ios.js
+├── android
+│   ├── app
+│   ├── build.gradle
+│   ├── gradle
+│   ├── gradle.properties
+│   ├── gradlew
+│   ├── gradlew.bat
+│   ├── keystores
+│   └── settings.gradle
+├── ignite
+│   ├── ignite.json
+│   └── plugins
+├── index.android.js
+├── index.ios.js
+├── ios
+│   ├── IgniteProject
+│   ├── IgniteProject-tvOS
+│   ├── IgniteProject-tvOSTests
+│   ├── IgniteProject.xcodeproj
+│   └── IgniteProjectTests
+└── package.json
+```
+
+## App directory
+
+Included in an Ignite project is the App directory. This is a directory you would normal have to create when using vanilla React Native.
+
+The inside of the App directory looks similar to the following:
+
+```
+App
+├── Components
+│   └── Styles
+├── Config
+├── Containers
+│   └── Styles
+├── Fixtures
+├── Images
+├── Lib
+├── Navigation
+│   └── Styles
+├── Redux
+├── Sagas
+├── Services
+├── Themes
+└── Transforms
+```
+
+**Components**
+"Dumb" components are stored here. All data is passed into dumb component. These components are often used inside a "Container Component". Container components are described in more detail further on.
+
+**Components ── Styles**
+We separate component styles from component functionality. Use this folder to create and store style files that matches the naming of your components. For example, a component `AlertMessage.js` would have a matching styles file called `AlertMessageStyles.js`.
+
+**Config**
+All application specific configuration falls in this folder. For example, `DebugSettings.js` is used for development-wide globals and `ReactotronConfig.js` is used for Reactotron client settings.
```

Comment:
`AppConfig.js` is for runtime configuration things.  (double check, @jamonholmgren may have renamed this to `index.js`.